### PR TITLE
Bulk operation

### PR DESCRIPTION
**Pull request by noopurmath3**

### CODE REVIEW

File: suites/test_bulk_operations.py [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+        bulk_edit_metadata = utilities.find_element(self.driver, By.XPATH, locators.BULK_EDIT_METADATA_XPATH)
+        bulk_edit_metadata.click()
+        search_beta_page.add_bulk_metadata(self.driver)
+        check_that("Alert message displayed when trying to add metadata to published title",
+            utilities.get_text(self.driver, By.CSS_SELECTOR, locators.ALERT_TITLE_CSS),
+            equal_to(constants.error_message_on_edit_metadata))
+        utilities.click_element(self.driver, By.XPATH, locators.CLOSE_DETAILS_XPATH)
+        utilities.wait(15)
+        lcc.log_info("End of tests, executing teardown")
+
```
Comment (anarang) [inline suites/test_bulk_operations.py:138]: @noopurmath3 please remove the extra blank lines. only one blank line is expected at the end of file. thanks.

**Review by anarang — APPROVED** [review]
LGTM! Approved, minor minor comment.

File: pages/search_beta_page.py [hunk 2]
```diff
@@ -37,6 +37,15 @@ def search_module_and_click(driver, title):
         lcc.log_info("It appears that the module was not found, please check your test data.")
         raise e
 
+def search_titles(driver, title):
```
[the remark this answers is not in the document]
Comment (noopurmath3) [reply pages/search_beta_page.py:40]: The existing method is search_module_and_click(), in this method we are only searching for the title, we do not want to click on the title to navigate to the document details page here

[the remark this answers is not in the document]
Comment (anarang) [reply pages/search_beta_page.py:40]: ack.

**Pull request merged**